src/library_pthread.js

```diff
@@ -1377,7 +1377,7 @@ var LibraryPThread = {
   $establishStackSpace: function(stackTop, stackMax) {
     _emscripten_stack_set_limits(stackTop, stackMax);
 #if STACK_OVERFLOW_CHECK >= 2
-    ___set_stack_limits(_emscripten_stack_get_base(), _emscripten_stack_get_end());
+    ___set_stack_limits(stackTop, stackMax);
```

Comment:
While you're adding comments, it might be good to add a comment here about the difference between `_emscripten_stack_set_limits` and `__set_stack_limits`.

Reply:
Done